canonical-web-and-design/microstack.run
# [microstack] Initial build of content

## Done

- Built microstack hompage based on the [design](https://github.com/canonical-web-and-design/web-squad/issues/1623) and [copy doc](https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit#)
- Added a single page of docs
- Added new GTM/GA codes

## QA

- Download this branch
- Go to http://0.0.0.0:8039/
- See that the copy matches the [copy doc](https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit#)
- See that the design matches the [design](https://github.com/canonical-web-and-design/web-squad/issues/1623)
- Gi ti http://0.0.0.0:8039/docs
- See that the docs look ok and largely are based on [this site](https://opendev.org/x/microstack/src/branch/master/README.md)

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1624

## Screenshot

![image](https://user-images.githubusercontent.com/441217/64843450-10fd7900-d5fd-11e9-9fdd-8eed5a170b34.png)
